src/venue/core.cljs

```diff
@@ -241,12 +241,13 @@
                (when (satisfies? IInitialise vm)
                  (initialise vm state)))
              (when (and old-vm (satisfies? IDeactivate old-vm))
-               (deactivate old-vm (:state old-vm-fixture)))
+               (deactivate old-vm (-> venue-cursor target :fixtures old-vm-id :state)))
```

Comment:
what is old-vm-id? could it not be a keyword (or non fn thing) and break the threading?